workflow: use Lix

lheckemann opened the pull request:
The Nix 2.9.0 prerelease previously pinned is too old to evaluate recent nixpkgs versions. My personal preference is to upgrade to Lix rather than a newer Nix version, hence this PR, but I mostly just care about getting updates again so feel free to close in favour of a different update :)

Run hasn't finished but it looks like it works, so far: https://github.com/lheckemann/minecraft.json/actions/runs/18480262926/job/52653453644

lheckemann committed:
workflow: use Lix

Ninlives committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/8b39dd562cbb1f72190f1f34ef5e82d783775ff5' (2025-08-09)
  → 'github:NixOS/nixpkgs/4212b48876ed2b99c3f951f000b2263dbc7333bf' (2025-10-13)

Ninlives committed:
Update manifest - Automated Commit

Ninlives commented:
Hi, just updated the workflow, I'm still using the nix installer as I'm more familiar with it. Please help me check if everything works fine, as I'm busy with my work these days and currently have no time to test, thanks!

Ninlives committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/4212b48876ed2b99c3f951f000b2263dbc7333bf' (2025-10-13)
  → 'github:NixOS/nixpkgs/3fb838fe4e9c50081199aa511a956808f19bdee9' (2025-10-14)

Ninlives committed:
Update manifest - Automated Commit

lheckemann commented:
Looks like it works, thanks!
